packages/mobile/src/screens/sign-on-screen/screens/CreateEmailScreen.tsx

```diff
@@ -53,6 +62,31 @@ export const CreateEmailScreen = (props: SignOnScreenProps) => {
     [dispatch, navigation]
   )
 
+  const {
```

Comment:
this all does feel quite complex, but i get the focus just being parity with web :)